easybuild/easyconfigs/s/snakemake/snakemake-7.32.3-foss-2022b.eb

```diff
@@ -0,0 +1,90 @@
+easyblock = 'PythonBundle'
+
+name = 'snakemake'
+version = '7.32.3'
+
+homepage = 'https://snakemake.readthedocs.io'
+description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('GitPython', '3.1.31'),
+    ('IPython', '8.14.0'),
+    ('PyYAML', '6.0'),
+    ('wrapt', '1.15.0'),
+    ('PuLP', '2.7.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('ConfigArgParse', '1.7', {
+        'checksums': ['e7067471884de5478c58a511e529f0f9bd1c66bfef1dea90935438d6c23306d1'],
+    }),
+    ('datrie', '0.8.2', {
+        'checksums': ['525b08f638d5cf6115df6ccd818e5a01298cd230b2dac91c8ff2e6499d18765d'],
+    }),
+    ('toposort', '1.10', {
+        'checksums': ['bfbb479c53d0a696ea7402601f4e693c97b0367837c8898bc6471adfca37a6bd'],
+    }),
+    ('amply', '0.1.6', {
+        'checksums': ['61421103ccf8e1066717115fe7917610d831d551c68d31a110876a5b6c78aea4'],
+    }),
+    ('smart-open', '6.3.0', {
+        'sources': ['smart_open-%(version)s.tar.gz'],
+        'checksums': ['d5238825fe9a9340645fac3d75b287c08fbb99fb2b422477de781c9f5f09e019'],
+    }),
+    ('connection-pool', '0.0.3', {
+        'sources': ['connection_pool-%(version)s.tar.gz'],
+        'checksums': ['bf429e7aef65921c69b4ed48f3d48d3eac1383b05d2df91884705842d974d0dc'],
+    }),
+    ('stopit', '1.1.2', {
+        'checksums': ['f7f39c583fd92027bd9d06127b259aee7a5b7945c1f1fa56263811e1e766996d'],
+    }),
+    ('reretry', '0.11.8', {
+        'checksums': ['f2791fcebe512ea2f1d153a2874778523a8064860b591cd90afc21a8bed432e3'],
+    }),
+    ('throttler', '1.2.2', {
+        'checksums': ['d54db406d98e1b54d18a9ba2b31ab9f093ac64a0a59d730c1cf7bb1cdfc94a58'],
+    }),
+    ('dpath', '2.1.6', {
+        'checksums': ['f1e07c72e8605c6a9e80b64bc8f42714de08a789c7de417e49c3f87a19692e47'],
+    }),
+    ('plac', '1.3.5', {
+        'checksums': ['38bdd864d0450fb748193aa817b9c458a8f5319fbf97b2261151cfc0a5812090'],
+    }),
+    ('yte', '1.5.1', {
+        'checksums': ['6d0b315b78af83276d78f5f67c107c84238f772a76d74f4fc77905b46f3731f5'],
+    }),
+    ('humanfriendly', '10.0', {
+        'checksums': ['6b0b831ce8f15f7300721aa49829fc4e83921a9a301cc7f606be6686a2288ddc'],
+    }),
+    ('nbformat', '5.9.2', {
```

Comment:
I see that `nbformat`, `jupyter-core` and `fastjsonschema` are new (they weren't  in `snakemake-7.18.2-foss-2021b.eb`). I'm wondering if we should try to allign the versions of these with those used in e.g. `jupyter-server`. In the current PR, the versions are slightly different than what was used for `jupyter-server` in the `2022b` toolchain. 

Maybe @boegel has an opinion on that?

Reply:
I don't think we can reasonable expect to align extension versions across easyconfigs using the same toolchain...

Reply:
Ok, in that case: all good to go with this PR! :)